packages/iov-bns-governance/src/governor.ts

```diff
@@ -61,19 +61,46 @@ export class Governor {
   }
 
   public async buildCreateProposalTx(options: ProposalOptions): Promise<CreateProposalTx & WithCreator> {
+    const kind: "bns/create_proposal" = "bns/create_proposal";
+    const commonProperties = {
+      kind: kind,
```

Comment:
This can also be inlined: `kind: "bns/create_proposal" as const,`

Reply:
Had to separate it out to avoid this really weird cast: `kind: "bns/create_proposal" as "bns/create_proposal";` but much nicer with `as const`.